examples/cartopy_convert.py

```diff
@@ -55,21 +55,21 @@
 
 ###############################################################################
 # Note that we could have easily done this with an EPSG code like so:
-crs_epsg = ccrs.epsg('3857')
-df_epsg = df.to_crs(epsg='3857')
-
-# Generate a figure with two axes, one for CartoPy, one for GeoPandas
-fig, axs = plt.subplots(1, 2, subplot_kw={'projection': crs_epsg},
-                        figsize=(10, 5))
-# Make the CartoPy plot
-axs[0].add_geometries(df_epsg['geometry'], crs=crs_epsg,
-                      facecolor='white', edgecolor='black')
-# Make the GeoPandas plot
-df_epsg.plot(ax=axs[1], color='white')
+# crs_epsg = ccrs.epsg('3857')
+# df_epsg = df.to_crs(epsg='3857')
+#
+# # Generate a figure with two axes, one for CartoPy, one for GeoPandas
+# fig, axs = plt.subplots(1, 2, subplot_kw={'projection': crs_epsg},
+#                         figsize=(10, 5))
+# # Make the CartoPy plot
+# axs[0].add_geometries(df_epsg['geometry'], crs=crs_epsg,
+#                       facecolor='white', edgecolor='black')
+# # Make the GeoPandas plot
+# df_epsg.plot(ax=axs[1], color='white')

```

Comment:
oops - no that was for debugging since I was on a plane and cartopy requires internet to do epsg lookups. just pushed it w/o these commented out